examples/dmd/wave_equation.cpp

```diff
@@ -558,6 +604,12 @@ int main(int argc, char *argv[])
 
             if (i % windowNumSamples == 0 && i < ts.size()-1)
             {
+                if(project_initial_condition)
+                {
+                    result_u = dmd_u[curr_window]->predict(ts[i]);
+                    cout << "Projecting solution for new window at " << ts[i] << endl;
+                    dmd_u[curr_window+1]->projectInitialCondition(result_u, ts[i]);
```

Comment:
For me to better understand the changes, I guess this projection is not necessarily relevant to the snapshot interpolation?

Reply:
Correct.  This is simply projecting the initial conditions in each subsequent window to come from the final conditions of the preceding window.  This makes the DMD solution be a prediction across the entire time domain, instead of a series of approximations each across only a single window.